opennav_docking/src/simple_charging_dock.cpp

```diff
@@ -166,8 +166,10 @@ geometry_msgs::msg::PoseStamped SimpleChargingDock::getStagingPose(
   return staging_pose;
 }
 
-bool SimpleChargingDock::getRefinedPose(geometry_msgs::msg::PoseStamped & pose)
+bool SimpleChargingDock::getRefinedPose(geometry_msgs::msg::PoseStamped & pose, std::string id)
```

Comment:
```suggestion
bool SimpleChargingDock::getRefinedPose(geometry_msgs::msg::PoseStamped & pose, std::string /*id*/)
```

Then remove the `(void) id;`